server/db/db.js

```diff
@@ -148,15 +148,15 @@ async function getActiveClients(pgClient) {
       'getActiveClients',
       `
         SELECT c.*
-	FROM clients c
-	INNER JOIN (
-	  SELECT DISTINCT client_id AS id
-	  FROM buttons
-	  WHERE is_sending_alerts AND is_sending_vitals
-	) AS b
-	ON c.id = b.id
-	WHERE c.is_sending_alerts AND c.is_sending_vitals
-	ORDER BY c.display_name;
+        FROM clients c
+        INNER JOIN (
+          SELECT DISTINCT client_id AS id
+          FROM buttons
+          WHERE is_sending_alerts AND is_sending_vitals
+        ) AS b
+        ON c.id = b.id
+        WHERE c.is_sending_alerts AND c.is_sending_vitals
+        ORDER BY c.display_name;
```

Comment:
I don't see any differences in this part of the code. could you explain what has changed?

Reply:
Hi Ahmad, the previous code had these lines indented with tab characters (unlike the rest of the codebase), but I've changed these lines to be indented with spaces (*like* the rest of the codebase). No logic has changed; just indentation :) .